grails-app/services/org/broadinstitute/orsp/IssueService.groovy

```diff
@@ -200,6 +200,10 @@ class IssueService {
         if (!extraPropertiesList.isEmpty()) {
             saveExtraProperties(issue, extraPropertiesList)
         }
+        issue.extraProperties.addAll(extraPropertiesList)
+        if (extraPropertiesList.find {it.name == IssueExtraProperty.PROJECT_REVIEW_APPROVED} != null ) {
```

Comment:
Nit: In groovy, non-null and non-empty are both truthy, so I don't think you need the `!= null` here:
```suggestion
        if (extraPropertiesList.find {it.name == IssueExtraProperty.PROJECT_REVIEW_APPROVED}) {
```

Reply:
Got it!